code/game/machinery/adv_med.dm

```diff
@@ -118,7 +118,7 @@
 				var/obj/structure/LB = L.buckled
 				LB.user_unbuckle_mob(user)
 			var/mob/M = G.affecting
-			if (M.client)
+			if (M && istype(M) && M.client)
```

Comment:
Since `M` is typed as mob, `if (istype(M) && M.client)` will suffice. `istype(M)` will return `FALSE` if `M` is `null`.